.github/workflows/semantic-pull-request.yml

```diff
@@ -1,5 +1,5 @@
 name: "Semantic PR"
-description: Ensure your PR title matches the Conventional Commits spec (https://www.conventionalcommits.org/).
+# description: Ensure your PR title matches the Conventional Commits spec (https://www.conventionalcommits.org/).
```

Comment:
Why is this commented out? 🤔

Reply:
With it present, the GitHub action refused to run; it didn't recognise the description field!